docs/docs/reference/configuration.md

```diff
@@ -379,6 +379,14 @@ Enable/Disable to produce large messages to Kafka. by default, it's configuratio
 * Type: `optional`
 * Default value: `false`
 
+#### `SINK_KAFKA_LINGER_MS`
+
+Defines the max interval the sink should we wait for the sink/producer buffer to fill
```

Comment:
just add "in milliseconds" after "max interval"

Reply:
Added.